pkg/truncindex/truncindex.go

```diff
@@ -14,6 +14,22 @@ var (
 	ErrAmbiguousPrefix = errors.New("Multiple IDs found with provided prefix")
 )
 
+type ErrNotExist struct {
```

Comment:
Please document the exported error type.

Nit: `ErrNotFound` would read better.